pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "metricflow"
-version = "0.204.0.dev7"
+version = "0.204.0.dev8"
```

Comment:
`0.204.0` was just released, this should be `0.204.1.dev0`?

Reply:
+1

I've got a tiny change I'd like to make for an 0.204.1 after this as well so 0.204.1.dev0 or similar sounds good to me.